### ISSUE TITLE
Support enable/disable paid bursting on Managed Disks

### ISSUE THREAD

**Issue by yuemlu** [Azure/azure-cli#15554]
**Resource Provider**

DiskRP

**Description of Feature or Work Requested**
We offer paid bursting where customer can pay for an activation fee to get higher burst perf target and permission to burst beyond available tokens. The disk can burst up to the max burst perf target at any point of time and charged for transactions that exceed available tokens. Customer can enable disk bursting when the disk is not attached to a VM or the attached to a stopped VM. Customer can disable disk bursting at runtime after 12 hours after enablement. Target release date: 12/1/2020
Support a new feature flag to surface paid bursting on Azure Disk cmdlets:
az disk create
az disk update
az disk show

**Minimum API Version Required**

Disk RP 2010-09-29 API version

**Swagger Link**

Expected to land on 11/1/2020
PR for the swagger
https://github.com/scottlzhang/azure-rest-api-specs/pull/3 

**Target Date**

12/1/2020

**Comment by yungezz** [comment]
planned in S179

**Comment by qwordy** [comment]
@yuemlu When will the PR be merged into official repo?

**Comment by yungezz** [comment]
blocked by service

**Comment by yuemlu** [comment]
@yungezz  The public swagger release date is 1/4/2021. Python SDK will be generated by then.

**Comment by houk-ms** [comment]
@yuemlu PR created https://github.com/Azure/azure-cli/pull/16702 to support this feature in AzureCLI.
